File: app/src/main/java/com/commit451/gitlab/model/rss/Thumbnail.java
package com.commit451.gitlab.model.rss;

import android.net.Uri;

import org.parceler.Parcel;
import org.simpleframework.xml.Attribute;
import org.simpleframework.xml.Root;

@Parcel
@Root(strict = false)
public class Thumbnail {
    @Attribute(name = "url", required = true)
    Uri url;

    public Thumbnail() {}

    public Uri getUrl() {
        return url;
    }
}

File: app/src/main/java/com/commit451/gitlab/model/rss/Thumbnail.kt
package com.commit451.gitlab.model.rss

import org.parceler.Parcel
import org.simpleframework.xml.Attribute
import org.simpleframework.xml.Root

@Parcel(Parcel.Serialization.BEAN)
@Root(strict = false)
class Thumbnail {
    @field:Attribute(name = "url", required = true)
    lateinit var url: String
}
